api/CSSPseudoElement.json

```diff
@@ -0,0 +1,307 @@
+{
+  "api": {
+    "CSSPseudoElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSPseudoElement",
+        "support": {
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": [
+            {
+              "version_added": false
```

Comment:
Otherwise, this looks great, but I did have one suggestion here (and repeated for the other Firefox entries): if the feature still exists behind a flag, then these `{ "version_added": false }` entries aren't necessary and can be removed.

Reply:
Done!